src/components/icons/uploadImage.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react"
```

Comment:
```suggestion
import React from "react"
```

Reply:
I _think_ importing `*` takes more than necessary and might have an effect on the resulting bundle (not sure here). Just importing `React` only works fine